### ISSUE TITLE
Ongoing project maintenance

### ISSUE THREAD

**Issue by nkabir** [lxdock/lxdock#109]
I've merged pull request #108. @robvdl will have more time in about a month to help. In the meantime, are there any LXDock-specific guidelines for publishing to PyPI and readthedocs.io? Are there any other items to manage for the LXDock development life cycle?

**Comment by hsoft** [comment]
To my knowledge, there isn't much to do other than manage incoming issues and PRs and develop new features to your liking.

**Comment by robvdl** [comment]
We'll figure the release process out in time I think, we need to have a good look at the current state first, when wast the last time there as a pypi release, is the project in a good state for a pypi release or do we still need to fix a number of things first, etc.

I will be available in a week or so... not a month, what I was saying is I have holidays in a weeks time, that last a month so plenty of time coming up.  The last week of work is a bit hectic for me because I work with systems that get really busy during Christmas.

**Comment by robvdl** [comment]
I also wanted to know what to do about the README list of authors (maintainers), it lists the two original authors and then a link to the contributors.  I think this comes up in the docs too and several other places.

The question is what to put there now, I can put my name in that list, do we want the original author names to remain there is upto them.  Should this be a list of active maintainers or past maintainers as well.

**Comment by hsoft** [comment]
sure, no problem. Whatever seems best to you.

**Comment by robvdl** [comment]
I was thinking maybe 2 headings, active maintainers and past or previous maintainers mostly so people know who to contact for the project, as well as have a link to the contributors in github as we have now.

**Comment by robvdl** [comment]
I might also be able to convince maybe two more devs to handle code reviews for me, maybe one person from work as well as a friend who is starting to use lxdock too as a developer.

**Comment by robvdl** [comment]
@hsoft I can't seem to add @johncurley to the group because I am member of the group but not owner.

I want him to help out with my code review queue.

**Comment by ellmetha** [comment]
@robvdl I've just invited him to the group. 😉 

Regarding the list of authors, feel free to change it to whatever seems right to you. But I think that a mention regarding the original authors of this project should be kept somewhere.

**Comment by robvdl** [comment]
Off course I would never remove the original authors, which is pretty much what I said :)

**Comment by shuhaowu** [comment]
Are there still on going maintenance to this project?

**Comment by robvdl** [comment]
Yeah it is, sorry I haven't been able to reply on IRC as of lately as I am getting quite bogged down with projects at work as well as outside of work.  I've been meaning to get PRs through but come weekends I've been pretty burnt out.  I will get them through and do want to keep this project going.

**Comment by robvdl** [comment]
I'm a little worried about the project now myself, as I too have started using docker-compose, I too have stopped using lxdock now. I had intentions to do some catching up over the Christmas break but I am always extremely busy being a senior dev at work and am finding I have far too much backlog of other unrelated work to actually work on this project. I might still have a go at it in the the next year to come, but the project is not looking in a good shape anymore with the lack of maintainers.

**Comment by shuhaowu** [comment]
I'm still using it on my downstream fork

**Comment by robvdl** [comment]
The reason why I was feeling a bit bad about it is because I started making changes around shares and had the intention to do an 0.5 release but that never really happened.

**Comment by shuhaowu** [comment]
If this project needs a maintainer, I can try to maintain this as I use it on a daily basis for several different use cases. There are a also a bunch of features I would like to add to this, beginning with some sort of fast module system that would allow more extensive functionality (like the X integration PR that I have open) that might not necessarily belong in the main code base.

That said, I also have very limited time and can't at this moment actively develop on the system. I could theoretically help with bug fixes and what not as I think I have a decent-ish understanding of the codebase now. 

Alternatively if no one else uses this project and only I use it, that's fine too as I'll maintain my fork for the foreseeable future.

**Comment by robvdl** [comment]
Hi @shuhaowu I missed this message in 2019, must have been too busy during the pandemic and I didn't see this, or I just forgot about it.

Anyway, see #184

I basically can't maintain the project, it's irrisponsible to just let it sit here and rot and give people the illusion the project still works (I doubt it does). I would archive it unless you want to take over.
